### PR TITLE
323 init account cli

### PR DESCRIPTION
**Pull request by econesag**

### CODE REVIEW

File: takeoff/takeoff_cli/lib/services/project_service.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class ProjectsService {
+  final TakeOffFacade _takeOffFacade;
+  ProjectsService(
+    this._takeOffFacade,
+  );
+
+  initAccount(String cloud, String email) async {
+    switch (cloud) {
+      case "gc":
+        _takeOffFacade.initGoogleCloud("eduard.conesa-guerrero@capgemini.com");
```
Comment (airherna) [inline takeoff/takeoff_cli/lib/services/project_service.dart:13]: initGoogleCloud is called with a hardcoded account instead of the email parameter

**Review by airherna — CHANGES_REQUESTED** [review]
Resolve comments

File: takeoff/takeoff_cli/lib/services/project_service.dart (hunk 1 above)
Comment (econesag) [reply takeoff/takeoff_cli/lib/services/project_service.dart]: Changed!

File: takeoff/takeoff_cli/lib/input/commands/init_command.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import 'package:args/command_runner.dart';
+import 'package:takeoff_cli/services/project_service.dart';
+
+class InitCommand extends Command {
+  final ProjectsService service;
+  @override
+  final name = "init";
+  @override
+  final description =
+      "Initialize the account which will use the selecter cloud provider.";
```
Comment (airherna) [inline takeoff/takeoff_cli/lib/input/commands/init_command.dart:10]: Add types to variable declaration

**Review by airherna — CHANGES_REQUESTED** [review]
Resolve comments

**Review by airherna — APPROVED** [review]

**Pull request merged**